macros/cross_db_utils/date_trunc.sql

```diff
@@ -2,9 +2,9 @@
   {{ return(adapter.dispatch('date_trunc', 'dbt_utils') (datepart, date)) }}
 {%- endmacro %}
 
-{% macro default__date_trunc(datepart, date) %}
+{%- macro default__date_trunc(datepart, date) -%}
     date_trunc('{{datepart}}', {{date}})
-{% endmacro %}
+{%- endmacro -%}
 
 {% macro bigquery__date_trunc(datepart, date) %}
```

Comment:
For cross-db support, you'll need to do the same thing on each of the adapter-specific macros, like this: 

```suggestion
{% macro bigquery__date_trunc(datepart, date) -%}
```

Again, I haven't applied this _everywhere_ it's necessary - just one example